gkeys/gkeys/email_reminders.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import print_function
+
+import os
+import re
+import smtplib
+import sys
+
+from string import Template
+
+from email.mime.text import MIMEText
+from snakeoil.demandload import demandload
+
+if sys.version_info[0] >= 3:
+    py_input = input
+    _unicode = str
+else:
+    py_input = raw_input
+    _unicode = unicode
+
+demandload(
+    "gkeys.base:Args",
+    "json:load",
+)
+
+class Emailer(object):
+    '''Send an email reminder about the status of the user's GPG key'''
+
+    def __init__(self, results, gkey):
+        uid = ''
+        if gkey.uid:
+            uid = gkey.uid[0]
+            match = re.findall(r'[\w\.-]+@[\w\.-]+', uid)
+            uid = ''
+            if match:
+                uid = match[0]
+        self.uid = uid
+        self.sub = []
+        for key in results:
+            print(results[key])
```

Comment:
Is this a debug print?  if so, you missed cleaning it up.

But also you didn't add the logger to the init parameters.  You should, that will let you convert this to a self.logger.debug().  With adding the logger to the class it will let you add other logger messages elsewhere where appropriate, especially where it detects any errors.